client/src/components/AuditAdmin/Progress/JurisdictionDiscrepancies.tsx

```diff
@@ -48,6 +48,19 @@ function getChoiceName(
   return choice.name
 }
 
+function displayDiscrepancy(val: string | number | undefined): string | number {
```

Comment:
Nit: this function is used to format both votes and discrepancies, so the name isn't quite accurate. Maybe `formatVoteCount`?